stream/cons.rkt

```diff
@@ -0,0 +1,18 @@
+#lang racket
+
+;kons instead of cons because racket already have a cons
+;but I really want my own cons...for fun and profit =D
+
+(define (kons a b)
+  (lambda (i)
+    (if (= i 1)
+        a
+        b)))
+        
+(define (kons-head k)
```

Comment:
do you still dislikes `car` and `cdr` ?

Reply:
yup =P. The explanation on SICP about the names was not enough to stimulate me to use them. And the thing where you compose them, like cardracardsadasda..whatever, I find almost impossible to understand =/ .